SonarQube.Client.Tests/SonarQubeService_GetHotspotAsync.cs

```diff
@@ -67,6 +70,8 @@ public async Task GetHotspot_Response_From_SonarQube()
     ""endOffset"": 41
   },
   ""status"": ""TO_REVIEW"",
+  ""line"": 32,
```

Comment:
Is the line field needed?
The test still asserts the line from textRange -- should we take it from here instead?